include/SFML/Window/Event.hpp

```diff
@@ -276,6 +276,17 @@ class SFML_WINDOW_API Event
     template <typename T>
     [[nodiscard]] const T* getIf() const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Check if current event type is not `Empty`
+    ///
+    /// \return True if current event type is not `Empty`
+    ///
+    ////////////////////////////////////////////////////////////
+    [[nodiscard]] operator bool() const
```

Comment:
```suggestion
    [[nodiscard]] explicit operator bool() const
```
Adding `explicit` here prevents conversions to `bool` outside of conditionals which is more in line with what we want. We only want this conversion happening inside a `while`.

Reply:
Didn't know `explicit` could be put on conversion operators. Tested it and looks good to me